features/overflow-clip-margin.yml

```
@@ -0,0 +1,9 @@
+name: Overflow clip margin
```

If we're reporting on just one property and there's no well-known concept its referring to, the feature name is probably the property itself (this is specific to CSS and is more art than science, sorry).

```suggestion
name: overflow-clip-margin
```